config/services.yaml

```diff
@@ -5,6 +5,10 @@ services:
   _defaults:
     public: true
     autowire: true
+    bind:
+      $jsonDir: '%kernel.project_dir%/public/json'
+      $moduleDir: '%kernel.project_dir%/var/tmp/modules'
```

Comment:
Are they really temporary files?

Reply:
(just a question ^^)

Reply:
Mostly yeah, it can be seen as cache, to retrieve modules/PrestaShop information needed to generate the json files.

Reply:
```suggestion
      $moduleDir: '%kernel.project_dir%/var/cache/modules'
```

maybe use var/cache instead if it's here for cache purposes.